### PR TITLE
Ensure 3D rendering options are disabled when adding a mesh layer

### LINKED ISSUE #34985: [mesh] The 3d renderer checkbox is unchecked by default with no effect
Add a mesh layer
Open the layer properties dialog
Enable the 3d view tab
The "Enable 3D renderer" is unchecked while sub items it should disable are enabled
image

### PR DESCRIPTION
**Pull request by DelazJ**
Fixes #34985 (seems that #34999 did not)

### CODE REVIEW

File: src/app/3d/qgsmeshlayer3drendererwidget.cpp [hunk 1]
```diff
@@ -37,6 +37,7 @@ QgsMeshLayer3DRendererWidget::QgsMeshLayer3DRendererWidget( QgsMeshLayer *layer,
   mWidgetMesh = new QgsMesh3dSymbolWidget( layer, this );
   mWidgetMesh->configureForDataset();
   layout->addWidget( mWidgetMesh );
+  mWidgetMesh->setEnabled( false );
```
Comment (DelazJ) [inline src/app/3d/qgsmeshlayer3drendererwidget.cpp:40]: An alternative could be to call [onEnabledClicked()](https://github.com/qgis/QGIS/blob/master/src/app/3d/qgsmeshlayer3drendererwidget.cpp#L84). Don't know which one is better/safer/less expensive...

Comment (vcloarec) [inline src/app/3d/qgsmeshlayer3drendererwidget.cpp:40]: thanks for reporting (again) this issue. Indeed, #34999 didn't fix the issue... 
Instead, I would add this line here at line 60:
https://github.com/DelazJ/QGIS/blob/e9f1f71451a6b90c6f872dec3583f44f8945f32b/src/app
The widget was not set disable if  3D renderer was not provided...
Thanks again.

**Review by vcloarec — APPROVED** [review]

**Comment by DelazJ** [comment]
@vcloarec is anything missing?

**Comment by vcloarec** [comment]
@DelazJ  approved!

**Comment by DelazJ** [comment]
Oh sorry. I forgot you do not have the commit right, yet. Let's wait for a committer. Thanks

**Comment by DelazJ** [comment]
I Think this one should be backported to ltr also but it might require #34999 backport first.

**Pull request merged**